sdk/cognitiveservices/azure-cognitiveservices-language-luis/azure/cognitiveservices/language/luis/authoring/tests/test_apps.py

```diff
@@ -0,0 +1,254 @@
+from azure.cognitiveservices.language.luis.authoring import LUISAuthoringClient
+from baseTest import *
+import unittest
+
+
+class AppsTests(unittest.TestCase):
+
+    @use_client
+    def test_list_applications(self, client: LUISAuthoringClient):
```

Comment:
Type annotation works after Python 3.6, and we need this test to work even with Python 2.7. You can do this to support all versions of Python:

```python
def test_list_applications(
    self,
    client, # type: LUISAuthoringClient
)
```